subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -3127,6 +3127,76 @@ static inline void event_ping_prep(struct ll_conn *conn)
 }
 #endif /* CONFIG_BT_CTLR_LE_PING */
 
+static inline void dle_max_time_get(const struct ll_conn *conn,
+				    u16_t *max_rx_time, u16_t *max_tx_time)
+{
+	u32_t feature_coded_phy;
+	u32_t feature_phy_2m;
+	u16_t rx_time;
+	u16_t tx_time;
```

Comment:
Both need to be initialized to zero, to avoid compiler warning

Reply:
initialization added